tools/cxbe/Logo.cpp

```diff
@@ -0,0 +1,188 @@
+// ******************************************************************
+// *
+// *  This file is part of Cxbe
+// *
+// *  This program is free software; you can redistribute it and/or
+// *  modify it under the terms of the GNU General Public License
+// *  as published by the Free Software Foundation; either version 2
+// *  of the License, or (at your option) any later version.
+// *
+// *  This program is distributed in the hope that it will be useful,
+// *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+// *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// *  GNU General Public License for more details.
+// *
+// *  You should have received a copy of the GNU General Public License
+// *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+// *
+// *  (c) 2002-2003 Aaron Robinson <caustik@caustik.com>
+// *  (c) 2021 Stefan Schmidt
+// *
+// *  All rights reserved
+// *
+// ******************************************************************
+#include "Xbe.h"
+#include <fstream>
+#include <stdexcept>
+#include <sstream>
+
+uint08 defaultXbeLogo[] =
```

Comment:
Why not a vector like you have when you import a custom logo?

Reply:
Mostly because `std::vector` cannot be constructed at compile time, and that bothers me despite its negligible performance impact. I'd use `std::array`, but I don't want to have to mess with the compiler flags, and it would require rewriting more parts of cxbe, too.